.factorypath

```diff
@@ -0,0 +1,44 @@
+<factorypath>
```

Comment:
what's the changes in this path used for?

Reply:
I don't remember making modifications to that file

Reply:
Should I erase it or make modifications?